nestjs/src/auth/auth-user.model.ts

```diff
@@ -56,7 +57,7 @@ export class AuthUser {
     this.isAdmin = admin;
     this.isHirer = hirer;
     this.githubId = user.githubId;
-    this.appRoles = [admin ? Role.Admin : Role.User];
+    this.appRoles = [admin ? Role.Admin : Role.User, this.isHirer ? Role.Hirer : null].filter(Boolean);
```

Comment:
let's use or not use `this` in both cases  
```suggestion
    this.appRoles = [admin ? Role.Admin : Role.User, hirer ? Role.Hirer : null].filter(Boolean);
```  
or  
```suggestion
    this.appRoles = [this.isAdmin ? Role.Admin : Role.User, this.isHirer ? Role.Hirer : null].filter(Boolean);
```